src/Runner.Common/JobServer.cs

```diff
@@ -16,24 +19,41 @@ public interface IJobServer : IRunnerService
     {
         Task ConnectAsync(VssConnection jobConnection);
 
+        void InitializeWebsocketClient(ServiceEndpoint serviceEndpoint);
```

Comment:
I am trying to think whether we can merge `InitializeWebsocketClient` with `ConnectAsync` since they are both doing some kind of setup before the JobServer can be used.

Reply:
...also I think we initialize this in jobrunner etc., places, where we don't need to create websocket. I think explicit method for websocket can help to ensure we kick start that only when we need to

Reply:
...also I think we initialize this in jobrunner etc., places, where we don't need to create websocket. I think explicit method for websocket can help to ensure we kick start that only when we need to